### ISSUE TITLE
Inputs need labels with content

### ISSUE THREAD

**Issue by oliverjam** [fac24/week5-bereket-minju#10]
https://github.com/fac24/week5-bereket-minju/blob/c89c27d8d5c8195d7de21acd87eeeeab88652078/src/App.jsx#L28-L35

You have added a `<label>` element here, but there's no text in it, so the user doesn't know what the input does. If you think it's visually obvious that it's a search input then you should at least add an `aria-label` so that users who cannot see it will also know what the input is for

**Comment by minju25kim** [comment]
added search label content for readability.
